setup.cfg

```diff
@@ -0,0 +1,56 @@
+[metadata]
+name = create-aio-app
+version = attr: create_aio_app.utils.versiontools.get_self_version
+description = The tool that helps quickly create a basis for aiohttp application
+long_description = file: README.md
+long_description_content_type = text/markdown
+platforms = 
+    POSIX
+author = Mykhailo Havelia
+author_email = misha.gavela@gmail.com
+license = MIT
+license_file = LICENSE
+url = https://github.com/aio-libs/create-aio-app
+project_urls = 
+    Source Code = https://github.com/aio-libs/create-aio-app
+    GitHub Issues = https://github.com/aio-libs/create-aio-app/issues
+    Documentation = https://create-aio-app.readthedocs.io/
+    CI: Travis = https://travis-ci.com/aio-libs/create-aio-app
+
+classifiers =
+    License :: OSI Approved :: MIT License
+    Intended Audience :: Developers
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Operating System :: POSIX
+    Development Status :: 4 - Beta
+    Framework :: AsyncIO
+
+keywords =
+    create-aio-app
+    cookiecutter
+    aiohttp
+
+
+[options]
+packages = find_namespace:
+include_package_data = True
+zip_safe = True
+python_requires = >=3.6
+setup_requires =
+    setuptools_scm
+    setuptools_scm_git_archive
+install_requires =
+    cookiecutter
+    click >= 7.0
+
+
+[check]
+restructuredtext = true
```

Comment:
Is this used by twine?

Reply:
Twine doesn't have restructuredtext flag. I think this is connected with `python setup.py check --restructuredtext` and now we can drop it.

Reply:
Have removed it.